docker-compose-base.yaml

```diff
@@ -80,7 +80,7 @@ services:
       - 2181:2181
 
   kafka:
-    image: wurstmeister/kafka:2.12-2.2.1
+    image: wurstmeister/kafka:2.13-2.8.1
```

Comment:
We were using an older version of the image, but I have now updated it to the latest available version https://hub.docker.com/r/wurstmeister/kafka/tags.